packages/fetchai/skills/ml_train/handlers.py

```diff
@@ -201,7 +201,7 @@ def _handle_data(self, ml_trade_msg: MlTradeMessage) -> None:
             )
             self.context.task_manager.enqueue_task(
                 MLTrainTask(
-                    skill_context=self.context, train_data=data[:2], epochs_per_batch=5,
+                    skill_context=self.context, train_data=data[:2],
```

Comment:
This should only receive the weights and the data and any configuration needed. No context!